src/ReTestItems.jl

```diff
@@ -449,21 +449,24 @@ function _runtests_in_current_env(
             end
         end
         Test.TESTSET_PRINT_ENABLE[] = true # reenable printing so our `finish` prints
+        # Let users know if tests are done, and if all of them ran (or if we failed fast).
+        # Print this above the final report as there might have been other logs printed
+        # since a failfast-cancellation was printed, but print it ASAP after tests finish
+        # in case any of the recording/reporting steps have an issue.
+        print_completion_summary(testitems; failedfast=(cfg.failfast && is_cancelled(testitems)))
```

Comment:
ah, so @Drvi this would be maybe too late already, since we hang before shutting down the workers?

Maybe we actually just want a log in `manage_worker`, here?:
https://github.com/JuliaTesting/ReTestItems.jl/blob/e9b773022d7242dbebe1989385d0a2a49bf0b79d/src/ReTestItems.jl#L695

We could basically say
```
@info "Finished all tests. Closing Worker $workerid."
```
?

Reply:
Added. I just added a log for each worker, so they don't have to coordinate to know if they're the last worker to complete, and also because i support any of them could be the one to hang here.